src/memory_pool.c

```diff
@@ -99,6 +135,15 @@ static umf_result_t umfPoolCreateInternal(const umf_memory_pool_ops_t *ops,
         goto err_pool_init;
     }
 
+    // Set default property "name" to pool if exists
+    for (int i = 0; i < UMF_DEFAULT_SIZE; i++) {
+        if (CTL_DEFAULT_ENTRIES[i][0] != '\0') {
```

Comment:
You should only call it for entries with matching name, not all.

Reply:
Now, should be fine